### DISCUSSION TITLE
Can you stop scan?

### DISCUSSION THREAD

**Discussion by cochcoder** [ThioJoe/YT-Spammer-Purge#1020] [Q&A] [answered]
I was wondering if it is possible for me to be able to stop a scan that is dangerously close to the API limit and still report the spam that was found? Any help or answers would be greatly appreciated, as I keep finding small YouTube channels with this issue and I can't report spam from the logs.

**Comment by ThioJoe** [comment, marked as answer]
It is not possible to stop a scan once started

**Reply by cochcoder** [reply]
Ok, thank you for letting me know :) .
